### ISSUE TITLE
Project Status?

### ISSUE THREAD

**Issue by lifeforced** [davmixcool/stake-pool-contract#1]
Looks like you've started something cool. Planning on continuing with it?

**Comment by davmixcool** [comment]
Yeah... I plan to continue development based on feedback from other devs. I have some crazy plans in mind though. 😁
